src/polyswarm/formatters/text.py

```diff
@@ -108,6 +108,8 @@ def artifact_instance(self, instance, write=True, timeout=False):
         output.extend(self.artifact(instance, write=False))
         if instance.failed:
             output.append(self._red('Status: Failed'))
+            if instance.failed_reason:
+                output.append(self._red(f'Failed Reason: {instance.failed_reason}'))
```

Comment:
```suggestion
                output.append(self._red(f'Failure Reason: {instance.failed_reason}'))
```

Reply:
Yes I wanted to use the word "Failed" because this goes after the line "Status: Failed", but you are right.